index.html

```diff
@@ -0,0 +1,168 @@
+                    <span class="material-symbols-outlined">
+                        gallery_thumbnail
+                    </span>
+                    <h3>CC Exhibition</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. architecto assumenda accusamus mollitia
+                        eveniet
+                        est ullam ipsa.</p>
+                </div>
+                <div class="card">
+                    <span class="material-symbols-outlined">
+                        forum
+                    </span>
+                    <h3>Forum</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. architecto assumenda accusamus mollitia
+                        eveniet
+                        est ullam ipsa.</p>
+                </div>
+                <div class="card">
+                    <span class="material-symbols-outlined">
+                        note_alt
+                    </span>
+                    <h3>Workshop</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. architecto assumenda accusamus mollitia
+                        eveniet
+                        est ullam ipsa.</p>
+                </div>
+                <div class="card">
+                    <span class="material-symbols-outlined">
+                        hub
+                    </span>
+                    <h3>CC Ignite</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. architecto assumenda accusamus mollitia
+                        eveniet
+                        est ullam ipsa.</p>
+                </div>
+            </div>
+            <a class="see-program-desktop">SEE THE WHOLE PROGRAM</a>
+            <a class="primary-btn">Join Digital Marketing Summit 2023</a>
+        </div>
+    </section>
+    <section class="speakers">
+        <div class="container">
+            <h2>Featured Speakers</h2>
+            <hr class="one">
+            <div class="speaker-list hideContent"></div>
+            <div class="show-more">
+                <a href="#/">MORE</a><span class="material-symbols-outlined">
+                    expand_more
+                </span>
+            </div>
+        </div>
+    </section>
+    <section class="desktop-partners">
+        <div class="container">
+
+            <h2>Partner</h2>
+            <hr>
+            <div class="social-media-logos">
+                <h3>mozilla</h3>
+                <h3>Google</h3>
+                <h3>Naver</h3>
+                <h3>airbnb</h3>
+                <h3>Facebook</h3>
+
+            </div>
+
+        </div>
+    </section>
+    <footer class="desktop-footer">
+        <div class="container">
+
+            <h2>Digital Marketing Summit</h2>
+            <div class="rights">
+                <p>2023 Digital Marketing.</p>
+                <p>Some Rights Reserved.</p>
+            </div>
+        </div>
+
+    </footer>
```

Comment:
- [ ] As mentioned by the previous reviewer please make sure your footer looks like` Cindy shin's design`. 

Your design:

<img width="1489" alt="Screenshot 2023-04-20 at 11 33 30 AM" src="https://user-images.githubusercontent.com/44798044/233289450-470d356c-bd6c-4730-b6a5-44b91f36cb38.png">

Cindy shin's design:

<img width="1022" alt="Screenshot 2023-04-20 at 11 32 59 AM" src="https://user-images.githubusercontent.com/44798044/233289371-321ea580-d28b-434a-b434-5608d6aa9c33.png">